src/api/spec/features/beta/webui/package_requests_spec.rb

```diff
@@ -1,23 +1,30 @@
 require 'browser_helper'
 
 RSpec.describe 'Package Requests' do
-  let(:user) { create(:confirmed_user, login: 'titan') }
-  let(:target_project) { create(:project_with_package, package_name: 'goal') }
+  let(:user) { create(:confirmed_user, login: 'henne') }
+  let(:sender) { create(:confirmed_user, login: 'kugelblitz') }
+  let(:receiver) { create(:confirmed_user, login: 'titan') }
+  let(:source_project) { create(:project_with_package, package_name: 'ball', maintainer: sender) }
+  let(:source_package) { source_project.packages.first }
+  let(:target_project) { create(:project_with_package, package_name: 'goal', maintainer: receiver) }
   let(:target_package) { target_project.packages.first }
-  let(:source_project) { create(:project_with_package, package_name: 'ball') }
   let(:other_target_project) { create(:project_with_package, package_name: 'package_2') }
```

Comment:
`other_target_project` can be removed, its not used anymore

Reply:
removed